packages/document/doc-tools-doc/modern.config.ts

```diff
@@ -140,11 +140,13 @@ function getNavbar(): NavItem[] {
       text: getI18nKey('guide'),
       link: '/guide/getting-started',
       activeMatch: '/guide/',
```

Comment:
I think you can restore the doc config for debug.

Reply:
Thank you. I have resubmitted the code and it seems that the submitted code still has this issue：
Some checks were not successful